common/test/views/support/AmpAdTest.scala

```diff
@@ -10,14 +10,14 @@ class AmpAdTest extends FlatSpec with Matchers {
      val sectionId = "sectionId"
      val result = AmpAdDataSlot(article(sectionId)).toString()
 
-     result should include(sectionId)
+     result should include(s"/$sectionId/")
```

Comment:
Perhaps these test results should be hardcoded to make it clear what the whole ad unit should be.

Reply:
Yep, good shout. I'll update.